partiql-ast/src/main/java/org/partiql/ast/v1/AstEnum.java

```diff
@@ -0,0 +1,8 @@
+package org.partiql.ast.v1;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public abstract class AstEnum extends AstNode {
+    public abstract int code();
```

Comment:
```suggestion
    public abstract int code();
    public abstract String name();
```

Reply:
This could be a follow-up

Reply:
Yeah -- I'll cut a followup PR since this PR is getting somewhat large. Tracked in https://github.com/partiql/partiql-lang-kotlin/issues/1610.

Reply:
Added `name` method in https://github.com/partiql/partiql-lang-kotlin/pull/1630.